erpnext/selling/report/payment_terms_status_for_sales_order/payment_terms_status_for_sales_order.py

```diff
@@ -92,16 +116,29 @@ def get_so_with_invoices(filters):
 
 	so = qb.DocType("Sales Order")
 	ps = qb.DocType("Payment Schedule")
+	cust = qb.DocType("Customer")
```

Comment:
Why do we need to do a join with the `Customer` and `Item` table, can't this information be retrieved from the `Sales Order` and `Sales Order Item Table` respectively?

Reply:
oh, I wasn't aware that these are available in `Sales Order` and `Sales Order Item`. I'll remove it.